pages/windows/comp.md

```diff
@@ -0,0 +1,32 @@
+# comp
+
+> Compare the contents of two files or sets of files.
+> Use wildcards (*) to compare sets of files.
+
+- Compare files interactively:
+
+`comp`
+
+- Compare two specified files:
+
+`comp {{path/to/file_1}} {{path/to/file_2}}`
+
+- Display differences in decimal format:
+
+`comp /d {{path/to/file_1}} {{path/to/file_2}}`
+
+- Display differences in ASCII format:
+
+`comp /a {{path/to/file_1}} {{path/to/file_2}}`
+
+- Display line numbers for differences:
+
+`comp /l {{path/to/file_1}} {{path/to/file_2}}`
+
+- Use a case-insensitive comparison:
```

Comment:
"Compare files case-insensitively" might sound more consistent with the previous descriptions.